app/controllers/api/v1/accounts/inboxes_controller.rb

```diff
@@ -27,6 +27,7 @@ def update
     return unless @inbox.channel.is_a?(Channel::WebWidget) && inbox_update_params[:channel].present?
 
     @inbox.channel.update!(inbox_update_params[:channel])
+    @inbox.update_working_hours(params.permit(working_hours: Inbox::OFFISABLE_ATTRS)[:working_hours]) if params[:working_hours]
```

Comment:
```suggestion

```

we don't need this anymore since we have the same line of code before the return statement